eido/conversion.py

```diff
@@ -51,17 +52,43 @@ def run_filter(prj, filter_name, plugin_kwargs=None):
     :param dict plugin_kwargs: kwargs to pass to the plugin function
     :raise EidoFilterError: if the requested filter is not defined
     """
+    # convert to empty dictionary if no plugin_kwargs are passed
+    plugin_kwargs = plugin_kwargs or dict()
+
     installed_plugins = pep_conversion_plugins()
     installed_plugin_names = list(installed_plugins.keys())
+    path = plugin_kwargs.get("path")
+
     if filter_name not in installed_plugin_names:
         raise EidoFilterError(
             f"Requested filter ({filter_name}) not found. "
             f"Available: {', '.join(installed_plugin_names)}"
         )
     _LOGGER.info(f"Running plugin {filter_name}")
     func = installed_plugins[filter_name]
-    plugin_kwargs = plugin_kwargs or dict()
-    func(prj, **plugin_kwargs)
+
+    conv_result = func(prj, **plugin_kwargs)
+
+    if path is not None:
+        # make out dir if it doesn't
+        # already exist
+        if not os.path.exists(path):
+            os.makedirs(path)
+
+        # loop through and write necessary
+        # files to disk
+        for out_file in conv_result:
+            with open(f"{path}/{out_file}", "w") as f:
+                f.write(conv_result[out_file])
+    elif verbose:
+        # loop through and print to
+        # standard out
+        for out_file in conv_result:
+            sys.stdout.write(conv_result[out_file])
+    else:
```

Comment:
@nsheff This `else: pass` seems redundant to me. Can it be removed? I can't think of another case for this.

Reply:
Sure.

it might just be to make it explicit, or maybe something else was there and got removed

Reply:
That was it -- passes now.